README.md

```diff
@@ -84,100 +84,102 @@ Check out the repository to find examples / tests for each feature.
 Safely access to an element:
 
 ```swift
-var myArray = [1, 2, 3]
-print(myArray[safe: 0]) // Optional(1)
-print(myArray[safe: 10]) // nil
+var array = [1, 2, 3]
+print(array[safe: 0]) // Optional(1)
+print(array[safe: 10]) // nil
```

Comment:
First method in the Array extension: https://github.com/tbaranes/SwiftyUtils/blob/67e22067971a4c10084affe1302dd8549a2d0d09/Source/ArrayExtension.swift#L12